scylla/tests/test_scylla.py

```diff
@@ -25,7 +25,7 @@ def test_instance_default_check(aggregator, db_instance, mock_db_data):
 @pytest.mark.unit
 def test_instance_additional_check(aggregator, db_instance, mock_db_data):
     # add additional metric groups for validation
-    additional_metric_groups = ['scylla.alien', 'scylla.sstables']
+    additional_metric_groups = INSTANCE_ADDITIONAL_GROUPS
```

Comment:
It might be helpful to make this a separate test - that is one test with a subset of the additional metrics as before, and this one which just adds everything.

Reply:
Yup, sounds good to have both cases.

Reply:
Addressed in e88d54a :+1: